contracts/interfaces/IAddressBook.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.10;
+
+import {IWhitelistModule} from "./IWhitelistModule.sol";
+import {IOtoken} from "./IOtoken.sol";
+
+interface IAddressBook {
+    function getOtokenImpl() external view returns (IOtoken otoken);
```

Comment:
shouldn't this one return an address?

Reply:
will be better to return address so the AddressBook module have no dependencies to other modules.

Reply:
It depends on our decision on the interface coding style, but I'll change everything to `address` type for now.